graf3d/eve7/inc/ROOT/REveElement.hxx

```diff
@@ -18,11 +18,7 @@
 
 #include <memory>
 
-#ifdef NLOHMANN_JSON_PROVIDES_FWD_HPP
 #include <nlohmann/json_fwd.hpp>
```

Comment:
I believe that only `#include <nlohmann/json.hpp>` should be used, as `nlohmann/json_fwd.hpp` is not always available, and we don't have control over when it's available or not if nlohmann_json is taken from the system. Other than that, seems fine.

Reply:
Yeah, maybe the build system should detect that early on and switch to builtin_json=On...

Reply:
No, for distros you cannot enable builtins, it *has* to be taken from the system.